pwa/src/templates/about/AboutTemplate.tsx

```diff
@@ -52,14 +45,11 @@ export const AboutTemplate: React.FC = () => {
         <Paragraph className={styles.description}>
           Voor OpenCatalogi hanteren we de archimate definitie met de aanscherpingen die vanuit de VNG worden meegegeven
           in hoofdstuk 2.1 van{" "}
-          <span
-            onClick={() =>
-              open(
-                "https://www.gemmaonline.nl/images/gemmaonline/0/09/GEMMA_Gegevenslandschap_-_Informatiearchitectuurprincipes_v1_0.pdf",
-              )
-            }
-          >
-            <Link icon={<ExternalLinkIcon />} iconAlign="start">
+          <span>
+            <Link href="https://www.gemmaonline.nl/images/gemmaonline/0/09/GEMMA_Gegevenslandschap_-_Informatiearchitectuurprincipes_v1_0.pdf">
+              <Icon className="utrecht-icon--conduction-start">
+                <ExternalLinkIcon />
+              </Icon>{" "}
```

Comment:
This used to trigger a `open()`, this way it opened in another tab. Please check all updated links, if they used to open in a new tab, they still should do so.

Reply:
Add `target="_new"`